Tests/UnitTests/Scripts/tests.js

```diff
@@ -225,6 +225,209 @@ describe("clearTimeout", function () {
     });
 });
 
+// URL
+describe("URL", function () {
+
+    // Currently all of the properties that the polyfill has implemented
+    function checkURL(
+        url,
+        href,
+        hostname,
+        origin,
+        pathname,
+        search) {
+        expect(url).to.have.property("hostname", hostname);
+        expect(url).to.have.property("href", href);
+        expect(url).to.have.property("origin", origin);
+        expect(url).to.have.property("pathname", pathname);
+        expect(url).to.have.property("search", search);
+    }
+
+    // Base URLs:
+    const baseUrl = 'http://httpbin.org';
+
+    it('should load URL with no pathname / search', function () {
+        // Standard URL (No pathname, no search)
+        let url = new URL(baseUrl);
+        // => 'http://httpbin.org'
+        checkURL(url,
+            'http://httpbin.org',  // href
+            'httpbin.org',          // hostname
+            'http://httpbin.org',  // origin
+            '',                               // pathname
+            ''                                // search
```

Comment:
nit: comments don't line up

Reply:
One thing you could do to avoid having to comment is to pass in an object instead of multiple parameters. Then you can write the code like this instead:
```javascript
checkUrl(url, {
    href: '...',
    hostname: '...",
    // ...
});
```